server/lib/txgh-server/triggers/handler.rb

```diff
@@ -45,6 +45,23 @@ def initialize(options = {})
         @logger = options[:logger]
       end
 
+      private
+
+      def update_github_status
+        Txgh::GithubStatus.update(project, repo, branch)
+      rescue Octokit::UnprocessableEntity
+        # raised because we've tried to create too many statuses for the commit
+      rescue Txgh::TransifexNotFoundError
+        # raised if transifex resource can't be found
+      end
+
+      def update_gitlab_status
+        Txgh::GitlabStatus.update(project, repo, branch)
+      rescue ::Gitlab::Error::Unprocessable
+        # raised because we've tried to create too many statuses for the commit
+      rescue Txgh::TransifexNotFoundError
+        # raised if transifex resource can't be found
```

Comment:
Same question about Rollbar

Reply:
No, octokit/gitlab error occurs when it's not possible to update commit status and it's fine because txgh is not the only thing which is messing with pipeline status. 
Regarding transifex error, it's also "kinda" expected because the resource may be missing there yet. We can change the flow to request all resources and first check if the needed one is there, but instead we are doing one direct GET request and rescue if it's not present, not ideal, but it's faster and is already working for few years now, so I didn't rewrite the flow for gitlab

Reply:
Sounds good, thanks for explaining!